src/consensus/aft/impl/aft_state.h

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
+
+#include "ds/logger.h"
+#include "ds/spin_lock.h"
+#include "kv/kv_types.h"
+#include "node/rpc/tx_status.h"
+#include "tls/key_pair.h"
+#include "tls/verifier.h"
+
+#include <map>
+#include <set>
+
+namespace aft
+{
+  class ViewHistory
```

Comment:
Is this duplicate of `TermHistory` in `raft.h`?

Reply:
TermHistory was renamed to ViewHistory to correctly match our naming convention

Reply:
Arg. I didn't see `raft.h` was changed - from the GitHub view, I thought it was only moved.